kpops/utils/pydantic.py

```diff
@@ -21,12 +22,79 @@ def to_dot(s: str) -> str:
     return s.replace("_", ".")
 
 
-class CamelCaseConfig(BaseConfig):
-    alias_generator = to_camel
-    allow_population_by_field_name = True
+def by_alias(field_name: str, model: BaseModel) -> str:
```

Comment:
perhaps swapping the parameters makes sense, wdyt?

Reply:
agreed